nixos/lib/test-driver/test-driver.py

```diff
@@ -344,6 +345,18 @@ def systemctl(self, q, user=None):
             )
         return self.execute("systemctl {}".format(q))
 
+    def require_active_unit(self, unit):
```

Comment:
We could probably rename this to `require_unit_state`, with an optional parameter defaulting to `active`.